contrib/build-push-ceph-container-imgs-arm64.sh

```diff
@@ -27,7 +27,7 @@ function build_and_push_latest_bis {
   # latest-bis is needed by ceph-ansible so it can test the restart handlers on an image ID change
   # rebuild latest again to get a different image ID
   make RELEASE="$RELEASE"-bis FLAVORS="${CEPH_RELEASES[-1]},centos-arm64,7" BASE_IMAGE=centos:7 build
-  docker tag ceph/daemon:"$BRANCH"-bis-"${CEPH_RELEASES[-1]}"-centos-7-arm64 ceph/daemon:latest-bis
+  docker tag ceph/daemon:"$BRANCH"-bis-"${CEPH_RELEASES[-1]}"-centos-arm64-7-${HOST_ARCH} ceph/daemon:latest-bis
```

Comment:
Actually, the whole function is not needed. Please remove `build_and_push_latest_bis` completely.
Thanks!

Reply:
Could please also fix line 40 with:

` source "$SCRIPT_DIR"/build-push-ceph-container-imgs.sh`

So the bash linter will be happy.

Reply:
Thanks for looking into this, let's remove the function, that should solve our problem.

Reply:
Don't give up 😀

Reply:
https://2.jenkins.ceph.com/job/ceph-container-build-push-imgs-arm64/22/console

Same result.  Image name is `master-52672ea-luminous-centos-arm64-7-aarch64` but function expects `master-52672ea-luminous-centos-7-aarch64`.

Reply:
Alright, don't bother with this anymore, I'll look into this with @ErwanAliasr1 next week once he returns from PTO. We might need to fix the build engine.
CI pipeline is more important 😀 .

Don't close the PR though.
Thanks.